rb/lib/selenium/webdriver/common/logger.rb

```diff
@@ -48,9 +48,13 @@ class Logger
       #
       # @param [String] progname Allow child projects to use Selenium's Logger pattern
       #
-      def initialize(progname = 'Selenium')
+      def initialize(progname = 'Selenium', ignored: nil)
         @logger = create_logger(progname)
-        @ignored = []
+        @ignored = Array(ignored)
+
+        warn("Details on how to use and modify Selenium logger:\n", id: [:logger_info]) do
```

Comment:
This will print the line on the first initialization of the logger, even if there are no warnings later in the code. Maybe we should instead print this once in the first call of `deprecate`?

Reply:
We don't initialize the logger by default, though, so it should only print if the logger actually prints a warning.

Hmm, though I guess it still prints if you initialize it and immediately set it to trace.

I'll rework.

Reply:
We initialize it all over the Selenium codebase too, so it is likely going to be printed as soon as you instantiate the driver.

Reply:
oh, right, the logger is the conditional... yup.

Reply:
ok, this should be better. :)